### PR TITLE
Add german translations

### PR DESCRIPTION
**Pull request by l0drex**
Hey,

I added some German translations to the application and to the metadata file.

### CODE REVIEW

**Comment by l0drex** [comment]
Also, I made a simple generator for metadata files over [here](https://gist.github.com/l0drex/d98790a5e4e6279fefcb9318ef6aab13). It evenly distributes the images over the day.

File: src/translations/po/de.po [hunk 1]
```diff
@@ -0,0 +1,247 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (zzag) [inline src/translations/po/de.po:1]: can you please add yourself to the copyright header? e.g.

```
# SPDX-License-Identifier: GPL-3.0-or-later
#
# SPDX-FileCopyrightText: 2021 name <email>
```

name can be a pseudonym if you're concerned about privacy. the most important part is email.

**Comment by l0drex** [comment]
I made the requested changes. Is the copyright notice correct? I was not sure what to replace there

**Comment by l0drex** [comment]
@zzag I think that worked. However, the branch I want to merge is still called master. I tried to rename it, but that closed this pull request (see above)

**Pull request merged**